### PR TITLE
Enforce bind-ip

### PR DESCRIPTION
**Pull request by hpihkala**
Check that the internal IP address is set when `streamr-docker-dev start`:ing services, otherwise do the `bind-ip` automatically. 

I've wasted so much time debugging mystery errors caused by forgetting `bind-ip`, especially on Travis... This would have helped.

### CODE REVIEW

File: streamr-docker-dev/bin.sh [hunk 1]
```diff
@@ -42,6 +42,11 @@ check_services_from_arguments() {
 }
 
 start() {
+    ip_lines=$( ifconfig | grep 10.200.10.1 | wc -l )
```
Comment (kare) [inline streamr-docker-dev/bin.sh]: ip_lines=$(ifconfig | grep -c 10.200.10.1)

File: streamr-docker-dev/bin.sh [hunk 2]
```diff
@@ -42,6 +42,11 @@ check_services_from_arguments() {
...
+    if [ "$ip_lines" -eq "0" ]; then
+       echo "WARNING: bind-ip is not set! Setting it now.";
```
Comment (kare) [inline streamr-docker-dev/bin.sh]: redundant ;

**Comment by hpihkala** [comment]
Fixed, thanks @kare!

**Comment by hpihkala** [comment]
It isn't only for MacOS

**Pull request merged**